src/Nri/Ui/Tooltip/V1.elm

```diff
@@ -0,0 +1,635 @@
+        case arrowPosition of
+            OnTop ->
+                [ ( "left", "50%" )
+                , ( "top", "calc(-" ++ String.fromFloat arrowSize ++ "px - 2px)" )
+                , ( "position", "absolute" )
+                ]
+
+            OnBottom ->
+                [ ( "left", "50%" )
+                , ( "bottom", "calc(-" ++ String.fromFloat arrowSize ++ "px - 2px)" )
+                , ( "position", "absolute" )
+                ]
+
+            OnLeft ->
+                [ ( "top", "50%" )
+                , ( "left", "calc(-" ++ String.fromFloat arrowSize ++ "px - 2px)" )
+                , ( "position", "absolute" )
+                ]
+
+            OnRight ->
+                [ ( "top", "50%" )
+                , ( "right", "calc(-" ++ String.fromFloat arrowSize ++ "px - 2px)" )
+                , ( "position", "absolute" )
+                ]
+
+
+pointerBox : Position -> Attribute msg
+pointerBox position =
+    css
+        [ Css.backgroundColor Colors.navy
+        , Css.border3 (Css.px 1) Css.solid Colors.navy
+        , arrowInPosition position
+        , Fonts.baseFont
+        , Css.fontSize (Css.px 16)
+        , Css.fontWeight (Css.int 600)
+        , Css.color Colors.white
+        , Global.descendants [ Global.a [ Css.textDecoration Css.underline ] ]
+        , Global.descendants [ Global.a [ Css.color Colors.white ] ]
+        ]
+
+
+viewCloseTooltipOverlay : msg -> Html msg
+viewCloseTooltipOverlay msg =
+    Html.button
+        [ css
+            [ Css.width (Css.pct 100)
+            , -- ancestor uses transform property, which interacts with
+              -- position: fixed, forcing this hack.
+              -- https://www.w3.org/TR/css-transforms-1/#propdef-transform
+              Css.height (Css.calc (Css.px 1000) Css.plus (Css.calc (Css.pct 100) Css.plus (Css.px 1000)))
+            , Css.left Css.zero
+            , Css.top (Css.px -1000)
+            , Css.cursor Css.pointer
+            , Css.position Css.fixed
+            , Css.zIndex (Css.int 90) -- TODO: From Nri.ZIndex in monolith, bring ZIndex here?
+            , Css.backgroundColor Css.transparent
+            ]
+        , Events.onClick msg
+        ]
+        []
+
+
+tooltipContainerStyles : List Style
+tooltipContainerStyles =
+    [ Css.display Css.inlineBlock
+    , Css.textAlign Css.left
+    , Css.position Css.relative
+    ]
+
+
+buttonStyleOverrides : List Style
+buttonStyleOverrides =
+    [ Css.cursor Css.pointer
+    , Css.border Css.zero
+    , Css.backgroundColor Css.transparent
+    , Css.fontSize Css.inherit
+    , Css.fontFamily Css.inherit
+    , Css.color Css.inherit
+    , Css.margin Css.zero
+    , Css.padding Css.zero
+    , Css.focus
+        [ -- TODO: Removing focus outline is poor accessibility practice
+          -- Consider fixing this by making focus outlines appear site-wide only on keyboard
+          -- navigation to meet design's needs -- hackday project, perhaps?
+          Css.outline Css.none
+        ]
```

Comment:
☁️ Did Hadley's outline PR make this no longer necessary?

Reply:
Yup, removed!